toolkit/tools/internal/randomization/randomization.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package randomization
+
+import (
+	"crypto/rand"
+	"fmt"
+	"math/big"
+	"strings"
+)
+
+// RandomString generates a random string considting of length runes
```

Comment:
>length [](start = 56, length = 6)

Now I'd either put "lenght" in quotes or maybe rephrase it a bit. Otherwise the first question that comes to mind is "what are 'length runes'?".:)